pull-request/src/person.js

```diff
@@ -7,9 +7,12 @@ function Person(first, last) {
 }
 
 Person.prototype.fullname = function() {
+    console.log("About to return full name");
     return this.firstName + " " + this.lastName;
 }
 
 Person.prototype.wishHappyBirthday = function() {
+    console.log("About to wish someone a happy birthday at age " + this.age + "!");
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9067/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9067/project/issues?id=training%3Apull-request&pullRequest=41&issues=AWgYV1bTUHOAr7vtan5Y&open=AWgYV1bTUHOAr7vtan5Y)

[ISSUE_KEY]: AWgYV1bTUHOAr7vtan5Y